FpsCheck.py

```diff
@@ -366,33 +364,48 @@ def find_invalid_well_known(self, check_sets):
             try:
                 json_schema = self.open_and_load_json(url)
                 curr_fps_set = check_sets[primary]
-                schema_fields = set({field for field in json_schema.keys()
-                                     if field in self.acceptable_fields})
-                schema_fields.update({field for field in self.acceptable_fields
-                        if len(curr_fps_set.relevant_fields_dict[field]) > 0})
-                for field in schema_fields:
-                    if field == "primary":
-                        if json_schema["primary"] != curr_fps_set.primary:
-                            field_sym_difference = [json_schema["primary"], 
-                            curr_fps_set.primary]
-                        else:
-                            field_sym_difference = []
-                    else:
-                        field_sym_difference = set(json_schema[field] 
-                                if field in json_schema.keys() else []) ^ set(
-                                    curr_fps_set.relevant_fields_dict[field])
-                        if field == 'ccTLDs':
-                            for aliased_site in json_schema[field]:
-                                field_sym_difference.update(
-                                    set(json_schema[field][aliased_site]) ^ 
-                                    set(
-                                    curr_fps_set.relevant_fields_dict[field]
-                                    [aliased_site]))
-                    if field_sym_difference:
-                        self.error_list.append("The following member(s) of " 
-                        + field + " were not present in both the changelist "
-                        + "and .well-known/first-party-set.json file: " + 
-                        str(sorted(field_sym_difference)))
+                well_known_set = FpsSet(
+                    json_schema.get('ccTLDs'), 
+                    json_schema.get('primary'), 
+                    json_schema.get('associatedSites'), 
+                    json_schema.get('serviceSites'))
+                if well_known_set.primary != curr_fps_set.primary:
+                    field_sym_difference = [well_known_set.primary, curr_fps_set.primary]
+                    self.error_list.append("The following primary was not "
+                    + "present in both the changelist and "
+                    + ".well-known/first-party-set.json file: "
```

Comment:
```suggestion
                    + f"{WELL_KNOWN} file: "
```

Same suggestion in the other places that duplicate the well-known filepath.